changelog.md

```diff
@@ -8,10 +8,32 @@ _Nothing Yet_
 **Added:**
 - bpk-svgs
   - Added 17 new icons
+  - alert--active
```

Comment:
Why are there changelog changes in this PR?

Reply:
it was supposed to go in a separate PR as I screwed up the changelog, I pushed everything in go so it's get merged faster